src/modes/hints/client/activate.js

```diff
@@ -85,6 +85,16 @@ const activators = {
   focusLink: (event, target) => {
     target.focus()
     return 'Reset'
+  },
+  yankLink: (event, target) => {
+    const copyText = copyEvent => {
+      copyEvent.clipboardData.setData('text/plain', target)
```

Comment:
Are you trying to copy the HTML element or the link text? If you're trying to copy the link text, try something like:

```javascript
if (target && target.href) copyEvent.clipboardData.setData('text/plain', target.href)
```